sal_interfaces/MTMount/MTMount_Events.xml

```diff
@@ -101,6 +101,362 @@
       <Count>1</Count>
     </item>
   </SALEvent>
+  <SALEvent>
+    <Subsystem>MTMount</Subsystem>
+    <EFDB_Topic>MTMount_logevent_controllerSettingsName</EFDB_Topic>
+    <Description>Settings file used in the low-level controller.</Description>
+    <item>
+      <EFDB_Name>settingsName</EFDB_Name>
+      <Description>Settings file name.</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>MTMount</Subsystem>
+    <EFDB_Topic>MTMount_logevent_azimuthControllerSettings</EFDB_Topic>
+    <Description>A selection of low-level controller settings for the azimuth axis.</Description>
+    <item>
+      <EFDB_Name>minCmdPositionEnabled</EFDB_Name>
+      <Description>Min command position software limit enabled.</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>maxCmdPositionEnabled</EFDB_Name>
+      <Description>Max command position limit enabled.</Description>
```

Comment:
Good catch.

I strongly prefer "command position limit" to "command position software limit", because I think "software" is unnecessary and potentially confusing (what other kind of command limit would there be)? So I changed them all to remove the "software".

I suspect the word "software" was a holdover from my documentation of the L1 limits, which at one point read "L1 software limit" but which I changed to "L1 (software) limit", to emphasize that "software" was a clarification. I probably did a copy/paste and forgot to remove "software" for the command limits in a few places.